doc/design/restore_existing.md

```diff
@@ -0,0 +1,17 @@
+Restore From Existing Backup
+======
+
+We want to restore an etcd cluster from existing backup that operator has saved before.
+
+We add a field in BackupPolicy:
+```Go
+type BackupPolicy struct {
+    // InitPV...
+    InitPV string `json:"initPV"`
```

Comment:
> How will user know that?

The backup Pod "foo-backup" will have a mount to a PVC "foo-pvc"; PVC will bind to PV "foo-pv". Owner of a cluster doesn't necessarily need access to operator.

> I think the user input is only the name of the cluster. Ideally they can just specify the cluster name as recovery identification.

I think this is totally possible and more user friendly.
Do you have a end user story in mind to better depict the whole thing?